docs/pages/BlogPosts.json

```diff
@@ -1,6 +1,39 @@
 {
   "year": "2022",
   "digests": [
+    {
+      "week": "6/3",
```

Comment:
we should probably decide on a date format, maybe 06/03? (and 05 below)?

Reply:
Sure!